include/SFML/Graphics/Rect.inl

```diff
@@ -73,31 +73,35 @@ height(static_cast<T>(rectangle.height))
 
 ////////////////////////////////////////////////////////////
 template <typename T>
-bool Rect<T>::contains(T x, T y) const
+constexpr bool Rect<T>::contains(T x, T y) const
 {
+    // Not using 'std::min' and 'std::max' to avoid depending on '<algorithm>'
+    const auto min = [](T a, T b){ return (a < b) ? a : b; };
+    const auto max = [](T a, T b){ return (a < b) ? b : a; };
```

Comment:
![image](https://github.com/SFML/SFML/assets/39244355/72319feb-0e14-4429-8dc9-12ca7d7d37f3)

lol (not upset, just having a laugh)